mos.yml

```diff
@@ -48,7 +48,7 @@ config_schema:
   - ["sw.initial_state", "i", 3, {title: "Initial state on power-on: 0 - off, 1 - on, 2 - restore last state, 3 - matches input if in toggle mode, otherwise off"}]
   - ["sw.auto_off", "b", false, {title: "Whether the switch should automatically turn OFF after turning ON"}]
   - ["sw.auto_off_delay", "d", 0, {title: "Delay for automatically turning OFF, in seconds"}]
-  - ["sw.power_state_led", "i", -1, {title: "Power state LED enabled"}]
+  - ["sw.power_state_led", "i", -1, {title: "Power state LED: -1 - unsupported by device, 0 - off, 1 - on"}]
```

Comment:
let's call this `sw.state_led_en`. -1 - unsupported, 0 - disabled, 1 - enabled

please update other places accordingly

Reply:
I can do, I only want to say that there are different status LEDs in the Shelly's. Power in the PlugS, Connect/Paired in the PlugS, Wifi in the Dimmer/RGBW. So I think we should be more precise with the naming.

Reply:
but this one will be under specific switch section, i think it's clear. we can have a setting under system settings for system led indication.